### PR TITLE
completed simplified chinese translation

### PR DESCRIPTION
**Pull request by ewong604**

### CODE REVIEW

File: app/Language/zh-CN/About.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+EOS;
+
+
+return [
+	'title'		 => 'About CodeIgniter',
+	'wiseTitle'	 => 'Wise Men',
+	'wiseText'	 => $wiseMenIntro,
+	'ackTitle'	 => 'Acknowledgements',
+	'ackText'	 => $hatsOffToThese,
+];
```
Comment (jim-parry) [inline app/Language/zh-CN/About.php]: The two blocks have been translated. Wouldn't the titles here (title, wiseTitle and ackTitle) get trasnslated too?

File: app/Language/zh-CN/Community.php [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	'forumText'		 => $forumCommunity,
+	'slackTitle'	 => 'Slack',
+	'slackText'		 => $slackCommunity,
+	'githubTitle'	 => 'Github',
+	'githubText'	 => $githubCommunity,
+	'forumButton'	 => 'Visit the forum',
+	'slackButton'	 => 'Visit Slack',
+	'ci4Button'		 => 'CodeIgniter 4',
+	'ci3Button'		 => 'CodeIgniter 3',
+];
```
Comment (jim-parry) [inline app/Language/zh-CN/Community.php]: Same comment here. Some of these neeed to be translated too, no?

Comment (ewong604) [reply app/Language/zh-CN/Community.php]: There's no chinese translation for unique English words such as CodeIgniter and ci3Button

File: app/Language/zh-CN/About.php (hunk 1 above)
Comment (ewong604) [reply app/Language/zh-CN/About.php]: There's no proper Chinese Word for ackTitle, AckText.

Comment (jim-parry) [reply app/Language/zh-CN/About.php]: wiseTitle & ackTitle English words, eg "Acknowledgements"

Comment (jim-parry) [reply app/Language/zh-CN/About.php]: only the values, not the keys

File: app/Language/zh-CN/Community.php (hunk 2 above)
Comment (jim-parry) [reply app/Language/zh-CN/Community.php]: ditto

**Pull request closed without merging**